mergePDF/search-n-merge.py

```diff
@@ -0,0 +1,48 @@
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+from fuzzywuzzy import fuzz, process
+from dateutil.parser import parse
+import pandas as pd
+import numpy as np
+import os, glob
+
+scope = ['https://spreadsheets.google.com/feeds', 'https://www.googleapis.com/auth/drive']
+creds = ServiceAccountCredentials.from_json_keyfile_name('client_secret.json', scope)
+client = gspread.authorize(creds)
+
+# Find a workbook by name and open the first sheet
+# Make sure you use the right name here.
+sheet = client.open("FLOCK Song List").sheet1
+
+# date = sheet.find(input('Enter date(M/D/YYYY): '))
+date = sheet.find('5/10/2020')
+
+# TODO: can we improve by allowing different/multiple date formats?
+# date = parse(input('Enter date: '), fuzzy=True)
+# print(sheet.col_values(1),sheet.col_values(date.col))
+
+# format_frame = pd.DataFrame(sheet.col_values(1))
+# songs = pd.DataFrame(sheet.col_values(date.col))
+
+extensions = ['.pdf','.mp3','guitar', '.vocal','.piano', '.1','.2','manibusan'] # to omit
+def remove_multiple_strings(cur_string, extensions): # function to remove extensions in matched strings. Improve match results
```

Comment:
There's a function for this already! It's always best practice to use an existing library implementation if it exists rather than writing our own. Please see [example](https://www.kite.com/python/examples/4300/os-remove-the-file-extension-from-a-filename)

Reply:
Also should we even remove extensions? Aren't we only looking for PDF files? I think we can remove extension for matching score only once we have determined its a PDF file. No point of trying to append an MP3 file to our PDF for example.